.github/workflows/openshift-ci/ehugonne-quay-robot-secret.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: github-secret
+data:
+  .dockerconfigjson: ewogICJhdXRocyI6IHsKICAgICJxdWF5LmlvIjogewogICAgICAiYXV0aCI6ICJaV2gxWjI5dWJtVXJjWFZoZVY5eWIySnZkRHBEVlZaWVRVMUZUalE1UjFGYU1rWXlXVFJXVFVkTVZFaElVMHhhUTFKTk9USlhWRFF3T1RKWk1rVTVNbE5QU2pNNE1qaFBNekpST1RSU00xQlVRVnBCIiwKICAgICAgImVtYWlsIjogIiIKICAgIH0KICB9Cn0=
```

Comment:
should not be in the PR...

Reply:
it is a read-only secret, I couldn't make the helm chart yaml configureable to not use pull-secret for a public image (in that case) butotherwise I'm getting timeouts with kind

Reply:
Hi @ehsavoie - not 100% sure about the use case, but the OpenShift CI infra provides a way to configure pull secrets, see https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/

Basically it can be stored in a vault and will be mounted to a conventional mountpoint, to be used via job configuration. Feel free to reach out for a working examples or additional info.

Reply:
Removed